### PR TITLE
PTX-2197 fix collect bundle for openshift/rke

### PR DESCRIPTION
**Pull request by thiguetta**
Signed-off-by: Thiago Gonzaga <thi_gonzaga@yahoo.com.br>

### CODE REVIEW

File: tests/common.go [hunk 1]
```diff
@@ -533,6 +516,11 @@ func PerformSystemCheck() {
 				expect(file).To(beEmpty())
 			}
 		})
+		ginkgo.JustAfterEach(func() {
```
Comment (rodrigc) [inline tests/common.go]: Take the changes for PerformSystemCheck() out of this PR and put it in a separate PR.

**Comment by rodrigc** [comment]
We will test this branch on a few jobs before merging

**Comment by thiguetta** [comment]
also here we can see that kubelet.log are now being generated for rke also https://jenkins.portworx.co/job/Torpedo/view/Torpedo%20master/job/tp-master-rke-basic-vol-driver/156/artifact/spawn/k8s/clusterGroup0/support/torpedo-tp-master-rke-basic-vol-driver-150-2/kubelet.log

**Comment by thiguetta** [comment]
tested again here https://jenkins.portworx.co/job/Torpedo/view/Torpedo%20master/job/tp-master-rke-basic-vol-driver/164/artifact/spawn/rke/clusterGroup0/support/
